lib/course.rb

```diff
@@ -0,0 +1,9 @@
+class Course
```

Comment:
Look in your editor for a setting and change your indentation to 2 spaces, not 4. 2 space indentation is standard for the Ruby language